runtime/onert/backend/train/ops/DepthwiseConvolutionLayer.h

```diff
@@ -50,6 +50,10 @@ class DepthwiseConvolutionLayer : public ::onert::exec::train::ITrainableFunctio
 
   // TODO Consider if these tensors should be built in TensorBuilder
   std::unique_ptr<BackPropTensor> _act_back_prop_output;
+
+  bool _use_padded_filter;
+  std::unique_ptr<Tensor> _padded_filter;
+  std::unique_ptr<Tensor> _filter_buffers;
```

Comment:
Tensors not managed by `TensorBuilder` cannot be supported dynamic shapes. Therefore, this change had to be reverted.